recipe/meta.yaml

```diff
@@ -11,32 +11,38 @@ source:
 
 build:
   number: 0
-  script: {{ PYTHON }} -m pip install . -vv
-  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
 
 requirements:
   host:
     - pip
-    - python >=3.6
+    - python
     - setuptools
+    - wheel
   run:
-    - python >=3.6
-    - toolz
-    - cons >=0.4.0
-    - multipledispatch
-    - etuples >=0.3.1
-    - logical-unification >=0.4.1
```

Comment:
dependencies are still required: https://github.com/pythological/kanren/blob/v1.0.3/setup.py#L18-L24

Reply:
Thanks for catching that!